.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,64 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }}'
+        runs-on: ubuntu-20.04
+        env:
+            SYMFONY_PHPUNIT_VERSION: 8
+            SYMFONY_DEPRECATIONS_HELPER: weak
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.2'
+                      dependencies: 'lowest'
+                    - php-version: '7.2'
+                    - php-version: '7.4'
+                      symfony-version: '^4.4'
+                    - php-version: '7.4'
+                      symfony-version: '5.0.*'
+                    - php-version: '8.0'
+
+
+        services:
+            mysql:
+                image: mysql:5.7
+                env:
+                    MYSQL_ROOT_PASSWORD: root
+                ports:
+                    - 3306:3306
+                options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Require Specific Symfony Version
+              if: ${{ matrix.symfony-version }}
+              run: composer require --no-update symfony/symfony:${{ matrix.symfony-version }}
```

Comment:
i would prefer to keep using flex for managing this. requiring symfony/symfony will hide it if we missed to declare the dependency on a specific symfony component.